packages/notebook-extension/src/celltaglist.tsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright (c) Jupyter Development Team.
+ * Distributed under the terms of the Modified BSD License.
+ */
+
+import { NotebookPanel } from '@jupyterlab/notebook';
+import React from 'react';
+//import PropTypes from 'prop-types';
+import { ReactWidget, VDomModel, VDomRenderer } from '@jupyterlab/apputils';
+import { CommandRegistry } from '@lumino/commands';
+
+interface IPropsFilterButton {
+  onClick: () => void;
+  sectionLabel: string;
+  buttonLabel: string;
+}
+
+export function FilterButtonComponent(props: IPropsFilterButton) {
+  return (
+    <div>
+      <p> {props.sectionLabel}</p>
+      <button type="button" onClick={props.onClick}>
+        {props.buttonLabel}
+      </button>
+    </div>
+  );
+}
+
+export class FilterButtonWidget extends ReactWidget {
```

Comment:
I am not quite sure if we need a new dedicated button widget class for each button?

It seems to do mostly the same thing as [`CommandToolbarButton`](https://github.com/jupyterlab/jupyterlab/blob/897418f0f25ffbbee3aa06a62770a53246823f85/packages/ui-components/src/components/toolbar.tsx#L869C14-L869C34).

But `CommandToolbarButton` takes care of many accessibility, reactivity and usability aspects already.

On possibility would be moving most of the `CommandToolbarButton` code into a new `CommandButton` in `ui-components` and make the only distinction of `CommandToolbarButton` inherit from it and apply the toolbar classes.

Reply:
@krassowski If we decide to split out a `commandButton`out of `CommandToolbarButton`I guess this should be done in another PR right ?

Reply:
Yes, another PR would be preferable